sdk/lib/ucrt/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+
+set(CMAKE_CXX_STANDARD 17)
+
+# Replace the old CRT include directory with the UCRT include directory
+get_property(INCLUDE_DIRS DIRECTORY . PROPERTY INCLUDE_DIRECTORIES)
+list(REMOVE_ITEM INCLUDE_DIRS "${REACTOS_SOURCE_DIR}/sdk/include/crt")
+set_property(DIRECTORY . PROPERTY INCLUDE_DIRECTORIES ${INCLUDE_DIRS})
+include_directories(${REACTOS_SOURCE_DIR}/sdk/include/ucrt)
```

Comment:
Well, for now I intend to keep it. At some point we will hopefully be able use the new headers globally, but that requires some work. The question is, whether we want to simply drop linking to msvcrt and instead link to ucrtbase. In that case things would be much simpler. It would simply require to install ucrtbase, if you want to run stuff on XP/2k3.